docs/content/docs/ide/Benchmarking.md

```diff
@@ -5,20 +5,35 @@ type: docs
 
 # Бенчмарки IDE
 
-Используем [Performance testing plugin](https://intellij-support.jetbrains.com/hc/en-us/articles/207241225-Performance-testing-plugin-in-PhpStorm).
+{{< hint info>}} Здесь содержиться информация, специфичная для Авито {{< /hint >}}
```

Comment:
`{{< avito >}}` ?

Reply:
It adds an English disclaimer but the other parts of document are in Russian. Btw I've changed the doc as you suggested.